doc/webhook-on-hostnetwork.md

```diff
@@ -0,0 +1,178 @@
+## Admission Webhooks over localhost
+* Use `DaemonSet`: The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster bootstrap.
```

Comment:
Not true, you still need a scheduler.  But you don't need the SDN to be working.